electron/index.cjs

```diff
@@ -66,7 +90,7 @@ const createTray = (window) => {
   ]);
 
   tray.on('click', () => {
-    win.maximize();
+    setWindowBounds(window);
```

Comment:
I'm not sure what Tray() should be in MacOS because I see no menu with "Show" label anywhere. If anyone could test it on Windows and Linux it would be great.